tests/unit/EntityInfo.spec.ts

```diff
@@ -416,3 +416,43 @@ describe('EntityInfo - Click Tests - Dissolutions', () => {
     expect(wrapper.emitted().notInGoodStanding).toEqual([['dissolve']])
   })
 })
+
+describe('EntityInfo - Click Tests - Business Summary', () => {
+  // mock the mixin to always return True
+  const AllowableActionsMixin: any = {
+    methods: {
+      isAllowed: () => true
+    }
+  }
```

Comment:
This is one way to do it (globally) and it's fine here.

You could also do this if you need to control the return value:
https://github.com/bcgov/business-filings-ui/blob/40a77d9928b713c0c6dee7da600ba6e1bd6d8fcd/tests/unit/FilingHistoryList.spec.ts#L369

Reply:
I like it!

Reply:
No need to change this.

The mixin already has unit tests that verify its response to feature flags. THIS component is one level up so it can just trust that the mixin is working correctly.

Reply:
Copy that, ty!